conf/application.conf

```diff
@@ -100,6 +100,13 @@ microservice {
       port = 8502
       url = "http://localhost:8502"
     }
+
+    email-verification {
+      host = localhost
+      port = 9891
+      url = "http://localhost:9891"
+      enabled = false
+	}
```

Comment:
We will require Base/QA config.
Not a blocker, more of a reminder :)

Reply:
Yep will raise the PR for the config change